core/src/main/resources/hudson/model/View/newJob_de.properties

```diff
@@ -20,6 +20,12 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-JobName={0} Name
-CopyExisting={0} kopieren
 NewJob={0} anlegen
+ItemType.validation.required=Bitte w\u00E4hlen Sie einen Element-Typen
+ItemName.label=Geben Sie einen Element-Namen an
+CopyOption.label=Kopieren von
+ItemName.validation.required=Dies ist ein Pflichtfeld. Bitte geben Sie einen g\u00FCltigen Namen an.
+# TODO Type to autocomplete
+CopyOption.placeholder=
```

Comment:
@imod  How would you translate "type to autocomplete" in a way that isn't worse than nothing?